frontend/public/locales/en-US/analytics.json

```diff
@@ -333,5 +333,41 @@
     "attributionSetConversionGoalInfo": "Conversion goal typically represent events that can embody a specific business objective. Common events include purchases, successful payments, registrations, and so on.",
     "attributionSetConversionWindowInfo": "Only events that occur within the conversion window, close to the time of the target conversion events, will be included in the analysis.",
     "attributionSelectTouchpointEventInfo": "Events to be included in the attribution analysis, e.g., click events for promotion spot, landing page visits."
+  },
+  "segment": {
+    "title": "User Segments",
+    "desc": "User Segments description",
+    "type": {
+      "userDone": "User has done",
+      "userNotDone": "User has not done",
+      "userIs": "User is",
+      "userIsNot": "User is not",
+      "doneInSequence": "User has done in sequence",
+      "userInGroup": "User in segment group",
+      "userNotInGroup": "User not in segment group"
+    },
+    "comp": {
+      "title": "Create Segment",
+      "desc": "Create a segment by selecting the users that shared the same attributes of behaviors within a period of time.",
+      "userSegmentSettings": "User segment settings",
+      "segmentName": "Segment name",
+      "segmentNameDesc": "Provide a name for the segment",
+      "segmentNamePlaceholder": "Input segment name",
+      "segmentDescription": "Description",
+      "segmentDescriptionDesc": "Provide a useful description for the segment",
+      "segmentDescriptionPlaceholder": "Input segment description",
+      "refreshMethod": "Refresh method",
+      "refreshMethodDesc": "Specify how you want to update the segment. For auto-refresh, please specify a schedule.",
+      "refreshMethodManual": "Manual",
+      "refreshMethodAuto": "Automatic",
+      "expirationSettings": "Expiration settings",
+      "expirationSettingsDesc": "Segment will become manual refresh after the expiration date, Setting an expiration date for a segment avoid unnecessary cost when an segment is not being used. You can always update it back to Automatic refresh method.",
```

Comment:
I think we'd better keep the refresh method as is. When segment schedule is expired, just disable the EventBridge rule instead of changing the refresh method to "Manual".

The description can be like "You can avoid unnecessary cost by setting expiration date for a segment after which the segment will stop auto refresh."

Reply:
@zxkane @luorobin-a2z  What do you think?

Reply:
Or we can change this description later?

Reply:
btw, I'm using the suggested approach in the backend implementation.

Reply:
The reason I want to transit the segment from auto to manual is that I think we should disable the eventbridge rule when the segment reach expiration date, but user should still be able to manually run the segment it they wish to.

Reply:
@jingnanl  I will update this description in frontend implementation if needed.

Reply:
Please modify the description. The backend implementation meets the requirement that @luorobin-a2z said. User can always run the segment from details page.